x-pack/plugins/security/server/routes/api/public/roles/get.js

```diff
@@ -42,10 +42,17 @@ export function initGetRolesApi(server, callWithRequest, routePreCheckLicenseFn,
       value: roleKibanaApplications.map(({ resources, privileges }) => {
       // if we're dealing with a global entry, which we've ensured above is only possible if it's the only item in the array
         if (resources.length === 1 && resources[0] === GLOBAL_RESOURCE) {
+          const reservedPrivileges = privileges.filter(privilege => PrivilegeSerializer.isSerializedReservedPrivilege(privilege));
```

Comment:
Should we return a `_transform_error` if we detect reserved privileges assigned at a space resource? If I understand the logic correctly, they will be deserialized as space feature privileges, which I wouldn't expect to happen

Reply:
Yup, I will make it so.